Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -211,8 +216,7 @@ CFTypeID CGContextGetTypeID() {
 
 #pragma region Global State - Lifetime
 static void __CGContextInitWithRenderTarget(CGContextRef context, ID2D1RenderTarget* renderTarget) {
-    context->_impl.renderTarget = renderTarget;
-
+    context->SetRenderTarget(renderTarget);
```

Comment:
>context->SetRenderTarget(renderTarget); [](start = 4, length = 39)

I still don't think this should be here; let's back out SetRenderTarget and we can land it with a full constructor once develop merges in.

What say you?